simulation/types/app.go

```diff
@@ -18,4 +20,16 @@ type App interface {
 	// object provided to it during init.
 	NewDefaultGenesisState() map[string]json.RawMessage
 	Cdc() codec.JSONCodec
+	GetAccountKeeper() AccountKeeper
+	GetBankKeeper() BankKeeper
+}
+
+type AccountKeeper interface {
+	GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
+	GetAllAccounts(ctx sdk.Context) []authtypes.AccountI
+}
+
+type BankKeeper interface {
+	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
+	GetAllBalances(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
 }
```

Comment:
The alternative to this is to make the simulator take more arguments, which doesn't seem desirable.

We should probably rename these later to more generic names. (pubkey getter, balance getter, etc.)